pyheif/reader.py

```diff
@@ -55,6 +55,34 @@ def close(self):
             del self._heif_handle
 
 
+class HeifContainer:
```

Comment:
I believe it is a good idea to move all classes to the `models.py` module since it can be used in the writer. It could be done in the current o next PR.

Reply:
I agree, but will save it for a later PR.